test/suites/fips/validate-fips.robot

```diff
@@ -0,0 +1,50 @@
+*** Settings ***
+Documentation       Tests related to FIPs Validation
+
+Resource            ../../resources/common.resource
+Resource            ../../resources/selinux.resource
+Library             Collections
+
+Suite Setup         Setup
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${USHIFT_HOST}      ${EMPTY}
+${USHIFT_USER}      ${EMPTY}
+
+
+*** Test Cases ***
+FIPs Validation
+    [Documentation]    Performs a FIPs validation against the host
+    Fips Should Be Enabled
+
+Microshift Fips Binary
+    [Documentation]    Performs a FIPs validation against the microshfit binary
+    Microshift Binary Should Dynamicly Link OpenSSL
+
+
+*** Keywords ***
+Setup
+    [Documentation]    Test suite setup
+    Check Required Env Variables
+    Login MicroShift Host
+
+Teardown
+    [Documentation]    Test suite teardown
+    Logout MicroShift Host
+
+Microshift Binary Should Dynamicly Link OpenSSL
+    [Documentation]    Check if Microshift binary is FIPs compliant.
+    ${stdout}    ${rc}=    Execute Command
+    ...    go tool nm /usr/bin/microshift | grep -i " _cgo.*DLOPEN_OPENSSL"
+    ...    sudo=True    return_rc=True
+    Should Be Equal As Integers    0    ${rc}
+
+Fips Should Be Enabled
+    [Documentation]    Check if FIPs is enabled on RHEL.
+    ${stdout}    ${rc}=    Execute Command
+    ...    bash -x fips-mode-setup --check
```

Comment:
added `bash -x` to troubleshoot why the periodic FIPs metal test [failed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_microshift/2534/pull-ci-openshift-microshift-main-metal-periodic-test/1721452727495561216/artifacts/metal-periodic-test/openshift-microshift-e2e-metal-tests/artifacts/scenario-info/el92-src@fips/log.html) with error:
```bash
FIPS mode is enabled.
Inconsistent state detected
```

Reply:
Is this comment still relevant?

Reply:
no